plugins/modules/elastic_endpoint_security.py

```diff
@@ -155,8 +195,10 @@ def main():
           results['pkg_policy_status'] = "No Integration Package found, Package Policy not created becans check_mode is set to true"
           results['changed'] = False
               
-      if integration_object['title'] == 'Endpoint Security' or integration_object['title'] == 'Prebuilt Security Detection Rules':
-        updated_pkg_policy_object = kibana.create_securityctrl_baseline_settings(pkg_policy_object)
+      #if integration_object['title'] == 'Endpoint Security' or \
+      #  integration_object['title'] == 'Prebuilt Security Detection Rules' or \
+      #  integration_object['title'] == 'Linux'  :
+      updated_pkg_policy_object = kibana.create_securityctrl_baseline_settings(pkg_policy_object)
```

Comment:
This seems out of place. Above we are checking for `check_mode` and `pkg_policy_object` before making any changes but this falls outside those checks.